metrics/cli/test/cli/updater/config/model/firebase_config_test.dart

```diff
@@ -0,0 +1,87 @@
+// Use of this source code is governed by the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+
+import 'package:cli/cli/updater/config/model/firebase_config.dart';
+import 'package:test/test.dart';
+
+// ignore_for_file: avoid_redundant_argument_values
+
+void main() {
+  group("FirebaseConfig", () {
+    const authToken = 'authToken';
+    const projectId = 'projectId';
+    const googleSignInClientId = 'googleSignInClientId';
+    const json = {
+      'auth_token': authToken,
+      'project_id': projectId,
+      'google_sign_in_client_id': googleSignInClientId,
+    };
+
+    test("throws an ArgumentError if the given auth token is null", () {
+      expect(
+        () => FirebaseConfig(
+          authToken: null,
+          projectId: projectId,
+          googleSignInClientId: googleSignInClientId,
+        ),
+        throwsArgumentError,
+      );
+    });
```

Comment:
🏷️ 
```suggestion
    },);
```
Please, apply this suggestion to all similar situations to be consistent with other tests.